### PR TITLE
fix(FreeDrawLayer): add onEnableMapDraw

### PR DESCRIPTION
**Pull request by yasabere**
affects: @rentpath/react-ui-core

Add onEnableMapDraw to freeDrawLayer

Allows for onEnableMapDraw call back to be fired when enabling draw mode in free draw

### CODE REVIEW

**Comment by tadjohnston** [comment]
Explanation?

File: packages/react-ui-core/src/Gmap/FreeDrawLayer.js [hunk 1]
```diff
@@ -81,12 +82,13 @@ export default class FreeDrawLayer extends PureComponent {
 
   @autobind
   enableMapDraw() {
```
Comment (grenti) [inline packages/react-ui-core/src/Gmap/FreeDrawLayer.js]: can you rename this `enableMapDraw` method to `handleShapeDrawing`? I just noticed that it seemed like the same as the prop and it's slightly confusing.

Comment (hernquist) [reply packages/react-ui-core/src/Gmap/FreeDrawLayer.js]: To Anye's point, it is potentially confusing.

**Comment by tadjohnston** [comment]
I don't think we need this anymore do we?

**Pull request closed without merging**